INSTALL.md

```diff
@@ -40,6 +40,13 @@ $> git clone https://github.com/savoirfairelinux/sous-chef
 $> docker-compose build
 $> docker-compose up
```

Comment:
I ran those two commands and when doing 'up', the database fails to initialize for me. Do you get the same error in a clean setup?

`db_1     | The latest information about mysql_install_db is available at`
`db_1     | https://mariadb.com/kb/en/installing-system-tables-mysql_install_db`
`db_1     | Initializing database`
`db_1     | mkdir: cannot create directory '/var/lib/mysql//mysql': Permission denied`
`db_1     | Fatal error Can't create database directory '/var/lib/mysql//mysql'`

It appears to be a mac specific issue as found on [Stack Overflow](http://stackoverflow.com/questions/37357315/mariadb-docker-container-on-os-x-cant-create-database-directory). I'm looking for a solution...